src/Communication/SAF.Communication.PubSub.Cde/SubscriptionInternal.cs

```diff
@@ -29,26 +32,26 @@ public override void Unsubscribe()
         {
             base.Unsubscribe();
             _subscriber.MessageEvent -= OnMessage;
-            _rawCallback = null;
+            _rawHandler = null;
         }
 
-        public void With(Action<string, TheProcessMessage> callback)
+        public void SetRawHandler(Action<string, TheProcessMessage> rawHandler)
```

Comment:
Setting the handler might be moved into the constructor of this class. Is this possible?

Reply:
When we pass an action into the constructor we cannot decide whether its the Handler or the RawHandler.